src/ng_link/ng_state.py

```diff
@@ -328,6 +329,217 @@ def initialize_attributes(self, input_config: dict) -> None:
             elif key == "showScaleBar":
                 self.show_scale_bar = val
 
+            elif key == "title":
+                self.title = val
+
+            elif key == "crossSectionOrientation":
+                self.cross_section_orientation = val
+
+            elif key == "crossSectionScale":
+                self.cross_section_scale = val
+
+            elif key == "projectionScale":
+                self.projection_scale = val
+
+            elif key == "layout":
+                self.layout = val
+
+            elif key == "position":
+                self.position = val

```

Comment:
@miketaormina it think it would be useful to set an additional else with a raise ValueError if the key is not in the available params. What do you think?

Reply:
I thought about this too, but then wondered if it is better failing silently as it does here. The current behavior is that, if the key is not one of the enumerated cases, nothing happens (which is actually what prompted this PR).

Another problem is that I can't seem to find a schema for the neuroglancer JSON to know what is allowed.

Since the Neuroglancer client will simply ignore keys in the JSON that it can't use (it does this with the `ng_link` field in the JSONs currently being made), anything can get put into the state without causing problems. Because of this, I thought about iterating through all the (key, value) pairs in `input_config`, but then I noticed how the class was updating the properties through decorated methods and didn't know how to do it.

One thing we could do:
```
else:
    self.__state[key] = val
```
and I think it would still fail silently if `key` isn't recognized by Neuroglancer, but would likely break if `key` is a valid key but `value` is of the wrong type, for instance.

Reply:
In that case, then let's leave it how it is right now. Thanks!